Gemfile

```
@@ -62,6 +62,10 @@ group :development do
   gem 'listen', '~> 3.0'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+
+  # Debug
+  gem 'debase', '0.2.2.beta10'
```

```suggestion
  gem 'debase', '0.2.2'
```

---

The VSCode debugger has a problem with v0.2.2. The developer recommends v0.2.2.beta10.

---

Do I need to install any dependencies in order to get this version to install on Ubuntu?

---

The [vscode-ruby](https://github.com/rubyide/vscode-ruby#install-ruby-dependencies) documentation suggests that byebug could work instead of debase.  What do you think about that? I'm concerned about not being able to install dependencies from the master branch when this is merged.
```suggestion
  gem 'byebug'
```

---

It requires debase gem. I will try again with 0.2.2. I remember that there was a problem.

---

It seems to be working this time. I will change it to v0.2.2.